index.html

```diff
@@ -0,0 +1,121 @@
+          <li><a href="about.html">Program</a></li>
+          <li><a href="about.html">Join</a></li>
+          <li><a href="about.html">Sponsor</a></li>
+          <li><a href="about.html">News</a></li>
+          <li class="connected"><div class="connected-div"><a href="about.html">Get Connected</a></div></li>
+       </ul>
+      </nav>
+      <nav class="mobile-nav">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+     </nav>      
+      <div class="introduction">
+        <p class="welcome-par">"Hello! Web Developers"</p>
+        <h1>Web Development<br>
+        <span class="h1-span">online courses</span></h1>
+        <div class="long-par-cont">
+            <p class="long-par">Here at Web-Development courses is everthing you need to know about various programming languages, data structures, design patterns, front-end/back-end libraries, to land your coding dream job.</p>
+        </div>
+        <h2 class="h2-header">
+            <span class="h2-span">Live courses Comming soon!</span><br>
+            2023.2.8(THUR) ~ 9(FRI)
+        </h2>
+        <p class="location-par">@[https://www.youtube.com/watch?v=q0Ex0l_LgRQ]</p>
+      </div>
+    </header>
+
+<main>
+ <section class="main-courses">
+  <h3 class="section1-title">Our courses</h3>
+  <span class="title-line"></span>
+  <div class="courses">
+    <div class="course">
+        <img class="html-img" src="./images/html icone.jpg" alt="html image">
+        <h3>Ultimate HTML5 course</h3>
+        <p>Everything you need to build fast and stunning websites with HTML5.</p>
+    </div>
+    <div class="course">
+        <img class="css-img" src="./images/cssimg.webp" alt="css image">
+        <h3>Ultimate CSS3 course</h3>
+        <p>Everything you need to build fast and stunning websites with CSS3.</p>
+    </div>
+    <div class="course">
+        <img class="javascript-img" src="./images/Js-icone.png" alt="javascript image">
+        <h3>JavaScript course</h3>
+        <p>Master JavaScript: Go from Novice to Professional programmer here.</p>
+    </div>
+    <div class="course">
+        <img class="react-img" src="./images/react-js-icone.png" alt="react image">
+        <h3>For Mastering React</h3>
+        <p>This course covers, the way forward to building fast, interactive web apps. </p>
+    </div>
+    <div class="course">
+        <img class="git-img" src="./images/git-last-img.png" alt="git image">
+        <h3>Ultimate Git course</h3>
+        <p>Everything you need to know to use Git & GitHub,work effectively as a team. </p>
+    </div>
+    <div class="course">
+        <img class="redux-img" src="./images/redux-icone.png" alt="redux image">
+        <h3>Ultimate Redux course</h3>
+        <p>Go from beginner to expert in a couple of hours in this short course redux.</p>
+    </div>
+  </div>
```

Comment:
- [ ] Your articles section is good and very similar to the template, but to make the HTML code more semantically correct, please use the `article` tag to wrap each article.

Reply:
Thank you for the feedback. I appreciate your suggestion to use the <article> tag for each article to improve the semantic correctness of my HTML code. I will make the necessary changes. 😁